polysemy-zoo.cabal

```diff
@@ -130,7 +130,7 @@ test-suite polysemy-zoo-test
       async >=2.2 && <3
     , base >=4.9 && <5
     , compact >=0.1.0.1
-    , constraints >=0.10.1 && <0.13
+    , constraints >=0.10.1 && <0.14
```

Comment:
Please bump the version in the `package.yaml` file instead of the `.cabal` file.